+Prefs/Empty.m

```diff
@@ -0,0 +1,38 @@
+classdef Empty < Prefs.Numeric
```

Comment:
Useful for backend.

Reply:
explain? why would an empty pref array not suffice?

Reply:
I want it to return values to functions like units and labels. I want it to feel like a pref that the program to touch without actually having innards.